Paco-iOS/NSObject+J2objcKVO.m

```diff
@@ -0,0 +1,217 @@
+//
+//  NSObject+J2objcKVO.m
+//  Paco
+//
+//  Created by northropo on 8/5/15.
+//  Copyright (c) 2015 Paco. All rights reserved.
```

Comment:
Please remove all Copyright notices.

It is copyright Google, Inc. but if we do that we should include the official boilerplate comment for Google Opensource projects. Better just to remove the copyright altogether.